webapp/components/ContentMenu/ContentMenu.vue

```diff
@@ -172,6 +172,23 @@ export default {
               icon: 'user-times',
             })
           }
+          if (this.resource.isBlocked) {
+            routes.push({
+              label: this.$t(`settings.blocked-users.unblock`),
+              callback: () => {
+                this.$emit('unblock', this.resource)
+              },
+              icon: 'user-plus',
```

Comment:
questionable if we want to use the same icons as the `mute/unmute` or if we should have similar, but slightly different ones